syft/pkg/cataloger/golang/licenses.go

```diff
@@ -123,7 +117,15 @@ func (c *goLicenses) getLicensesFromRemote(moduleName, moduleVersion string) ([]
 
 	proxies := remotesForModule(c.opts.Proxies, c.opts.NoProxy, moduleName)
 
-	fsys, err := getModule(c.progress, proxies, moduleName, moduleVersion)
+	prog := monitor.StartCatalogerTask(monitor.GenericTask{
+		Title: monitor.Title{
+			Default: "Downloading go mod",
+		},
+		HideOnSuccess: true,
+		ParentID:      "TODO", // TODO: setting this to non-empty will cause the progress bar to be nested, but this needs to be more specific
```

Comment:
yeah, I need to remove the value, but this will be taken care of fully in #1383 . The impact is that this will be another sibling task and not nested under the "cataloging packages" task.